packages/mantine/src/components/modal-wizard/ModalWizardStep.tsx

```diff
@@ -21,7 +21,7 @@ export interface ModalWizardStepProps {
     /**
      * A tooltip label for the docLink
      */
-    docLinkTooltipLabel?: string | ReactElement | DependsOnStep<string | ReactElement>;
+    docLinkTooltipLabel: string | ReactElement | DependsOnStep<string | ReactElement>;
```

Comment:
Same reasoning here, will all ModalWizard usages have a doc link? I am not convinced

Reply:
I think there are two ways to solve this:

- Make the `docLinkTooltipLabel` required in ModalWizard to avoid the black icon ever show up in case people forget about passing in this prop when using ModalWizard
- Keep `docLinkTooltipLabel` optional

I am currently going with option 2, let me know what you think?

Reply:
The way I see it is that the issue of the black icon should be fixed, even if no tooltip label is provided

Reply:
I did some tests locally and it seems that the black icon will show up even if null/ empty is passed to the mantine tooltip component

Reply:
So what will be something that I can do from my end?

Reply:
It's not a black icon, it's the tooltip rendering without any text in it 🙈 

You can make the rendering conditional depending if a label is provided or not I think 👍

Reply:
It's purple not black :p 

![image](https://user-images.githubusercontent.com/63734941/208928117-de071f9e-6c6d-4d0c-b771-3c0dcbcef54a.png)

Reply:
Can't we modify the Header component implementation to avoid the issue? aka not rendering a Tooltip component if the label is empty.

Reply:
I think this makes sense, no need to render a component if it is missing some information, in this case, no need to show Tooltip if the tooltip does not have a label. I will test it out on the header component.

Reply:
> It's not a black icon, it's the tooltip rendering without any text in it 🙈
> 
> You can make the rendering conditional depending if a label is provided or not I think 👍

The Tooltip component requires a label, so I cannot conditionally the label 🤔

Reply:
Have you tried the `disabled` prop? 😉 

![image](https://user-images.githubusercontent.com/35579930/209003420-f2807ce8-ea7d-40f4-ac19-b9b3dcd32e2a.png)

Reply:
I decided to use `disabled` prop and it seems to be working well.

Reply:
> Can't we modify the Header component implementation to avoid the issue? aka not rendering a Tooltip component if the label is empty.

I tested this and it also works. so between these two approaches, I prefer to not render `Tooltip` at all when the label is missing, it does not make sense to render something in UI where it does not have a purpose, let me know what you think!